runtime/src/iree/hal/string_util.c

```diff
@@ -16,6 +16,7 @@
 
 #include "iree/base/api.h"
 #include "iree/base/internal/math.h"
+#include "iree/base/status.h"
```

Comment:
not needed - iree/base/api.h pulls in all iree/base/ headers

Reply:
ah yeah, the auto header includes are an annoying default feature in vscode/clangd. I disable it when I think about it, but here I'm on a different machine for the GPU.